pkg/model/proto/model.proto

```diff
@@ -172,6 +173,8 @@ message CreateVolumeOpts {
     int64 snapshotSize = 15;
     // Down load snapshot from cloud
     bool snapshotFromCloud = 16;
+    // The Serialized profile
+    string profile = 17;
```

Comment:
@Shruthi-1MN Please take a look at here.

Reply:
oh ok this is perfect.